src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -467,6 +467,58 @@ ucs_status_t uct_cuda_ipc_iface_init_streams(uct_cuda_ipc_iface_t *iface)
     return UCS_OK;
 }
 
+static ucs_status_t
+uct_cuda_ipc_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr)
+{
+    const double latency  = 1.8e-6;
+    const double overhead = 4.0e-6;
+
+    perf_attr->bandwidth.dedicated = 0;
+
+    if (perf_attr->field_mask & UCT_PERF_ATTR_FIELD_BANDWIDTH) {
+
+        /* TODO:
+         *      1. differentiate read vs write perf
+         *      2. check if src,dst memory is cuda type;
+         *         (currently UCP does not pass mem types so cuda_ipc would
```

Comment:
it does in protov2:
https://github.com/openucx/ucx/blob/4f8240871d11bc53bbb609263d32d0127be0dc57/src/ucp/proto/proto_init.c#L424

Reply:
@yosefe This seems to be true only for proto_init_buffer_copy_time function. Is this function used for RMA transfers or just local memtype_ep transfers (like pack/unpack)?

 I noticed that cuda-ipc perf estimate (most likely from zcopy estimation_perf logic) didn't provide memory types and cuda_ipc_estimate_perf was returning default performance. Examples include:
1. https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_ep.c#L1976
2. https://github.com/openucx/ucx/blob/master/src/ucp/proto/proto_common.c#L302
3. https://github.com/openucx/ucx/blob/master/src/ucp/rndv/proto_rndv.c#L211

Reply:
right, it's missing for the actual transfer which is relevant for cuda-ipc - in ucp_proto_common_get_lane_perf
place (1) is protov1 and place (3) is rndv control message that is always host mem.